src/components/select.js

```diff
@@ -49,6 +49,12 @@ class Select extends React.Component {
     values: []
   }
 
+  componentDidMount() {
```

Comment:
Since we need to do this here, I think it's better to also call `this.focus()` here, instead of passing the autofocus down to the input.